### ISSUE TITLE
Blazor Runtime Culture

### ISSUE THREAD

**Issue by grahamehorner** [aspnet/Blazor#60]
Please consider browser runtime cultural and allow switching of culture with in the Blazor environment

**Comment by SteveSandersonMS** [comment]
Good idea. It will be a while before we get to it, but it's good to have on the backlog.
